main/zoe/guide/contracts/barter-exchange.md

```diff
@@ -0,0 +1,17 @@
+<Zoe-Version/>
+
+##### [View the code on Github](https://github.com/Agoric/agoric-sdk/blob/958a2c0a3dec38bdba2234934119ea2c28958262/packages/zoe/src/contracts/barter-exchange.js) (Last updated: 6/30/2020)
```

Comment:
Good to see the addition of BarterExchange. Are there plans to also add multipoolAutoswap, automaticRefund, mintAndSellNFT, and sellItems?  I  would consider renaming the description of second-price auction to be publicAuction, to match the code.